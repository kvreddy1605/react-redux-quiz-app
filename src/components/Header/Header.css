.Header{
    background-color: #005960;
    color: white;
    font-size: 40px;
    text-align: center;
    padding: 20px;
}

.Header img{
    background-color: white;
    height: 34px;
    margin: 0px 10px;
}