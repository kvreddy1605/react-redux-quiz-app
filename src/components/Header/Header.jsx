import React from 'react';
import classes from './Header.css';
import image from '../../images/result.png'

const Header = (props) => {
    
    return(
        <div className={classes.Header}>
           <span>Assessment</span>
           <img src={image} alt={'assessment-img'}></img>
        </div>
    );

}

export default Header;