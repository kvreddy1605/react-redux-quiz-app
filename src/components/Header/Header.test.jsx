import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../Header/Header'


describe('header Component Test', () => {
    const props = {};
    const Component = <Header {...props}/>
    it('should match latest snapshot', ()=>{
        const snapshot = renderer.create(Component)
        expect(snapshot.toJSON()).toMatchSnapshot()
    })
})