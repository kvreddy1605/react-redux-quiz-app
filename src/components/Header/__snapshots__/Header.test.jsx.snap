// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header Component Test should match latest snapshot 1`] = `
<div
  className={undefined}
>
  Assessment
</div>
`;
